### DISCUSSION TITLE
Can I specify a POST ID on a specific post which caused duplicate generated indexes ?

### DISCUSSION THREAD

**Discussion by dimaslanjaka** [hexojs/hexo#5278] [Question / Answer]
when running hexo v7 twice without `hexo clean`, index generated duplicate orders.

![image](https://github.com/hexojs/hexo/assets/12471057/d69cbaa7-753e-47a5-b720-bdbad5eb9628)

I think this caused by POST ID always newly generated, and the post count are doubled like below screenshots

![image](https://github.com/hexojs/hexo/assets/12471057/60646514-dda4-4746-a16c-f4314aee9054)

> original posts only around 1.4k

I think this is an issue, but idk how to reproduce it. And I think maybe it's a bug of the theme I'm using (hexo-theme-butterfly).

How to fix this ?

my site repo at https://github.com/dimaslanjaka/static-blog-generator-hexo/tree/master/site

**Comment by dimaslanjaka** [comment]
@jerryc127 @yoshinorin

**Comment by stevenjoezhang** [comment]
I cannot reproduce this issue.

**Reply by dimaslanjaka** [reply]
maybe when i running same hexo directory on different CI, might caused this issue. now I changed other CI to trigger main CI on main repo.

previous CI:

https://github.com/dimaslanjaka/source-posts/blob/a8523d223e1e22ed5e9d8ba956a7ce5d98496764/.github/workflows/post-update.yml

which used reusable workflow of main repo https://github.com/dimaslanjaka/static-blog-generator-hexo/blob/master/.github/workflows/build-site.yml

the problem conflict at different cache image which caused duplicated ID
